packages/ember-intl/addon/-private/formatjs/format-date-time-range.ts

```diff
@@ -0,0 +1,12 @@
+import type { IntlShape } from '@formatjs/intl';
+
+export type FormatDateTimeRangeParameters = Parameters<
+  IntlShape<string>['formatDateTimeRange']
+>;
+
+export function formatDateTimeRange(
+  intlShape: IntlShape,
+  ...[from, to, formatOptions]: FormatDateTimeRangeParameters
+): string {
+  return intlShape.formatDateTimeRange(from, to, formatOptions);
```

Comment:
Can you check whether `@formatjs/intl@v3` uses positional or named arguments for `formatDateTimeRange`?

While I personally prefer named arguments (with names `from` and `to`) since there is more than 1 input, I also think it may be better for now to follow the same conventions as `formatjs`.

Reply:
These are the docs for `formatDateTimeRange`: https://formatjs.github.io/docs/react-intl/api/#formatdatetimerange

The signature is
```ts
function formatDateTimeRange(
  from: number | Date | string,
  to: number | Date | string,
  options?: Intl.DateTimeFormatOptions & {format?: string}
): string
```

If meant the react package of formatjs, then it uses named arguments:
```tsx
<FormattedDateTimeRange
  from={new Date('2020-1-1')}
  to={new Date('2020-1-15')}
/>
```

But this is because (afaik) react components don't support positional arguments. Even with a simple formatted date, they use them
```tsx
<FormattedDate
  value={new Date(1459832991883)}
  year="numeric"
  month="long"
  day="2-digit"
/>
```

In the case of ember-intl, to be consistent with the other helpers, I think it makes sense to have dates as positional and formatting options as named. I am open to suggestions, of course.

Reply:
To clarify, I was wondering if `intlShape.formatDateTimeRange()`, in `v3`, still accepts `from` and `to` as positional inputs/arguments, instead of named (via a POJO).

According to its types, it still expects positional arguments, so let's define our helper to expect positional arguments as well.